frontend/src/common/constansts/headers.ts

```diff
@@ -1,10 +1,5 @@
-export const HEADER_MAIN_TABS: string[] = [
-    "Home",
-    "Top Up",
-    "Transactions",
-];
-
-export const HEADER_SETTINGS_TABS: string[] = [
-    "Profile",
-    "Log out"
-];
```

Comment:
roll it back and instead of strings use constants

Reply:
If you don't understand, ask.  You are adding and removing again and again. 
You have to instead of strings use constants. 
```typescript
export const LOG_OUT_TAB = "Log out";

export const HEADER_MERCHANT_SETTINGS_TABS: string[] = [
    LOG_OUT_TAB
];

export const HEADER_CLIENT_SETTINGS_TABS: string[] = [
    PROFILE_TAB,
    LOG_OUT_TAB
];

```